### ISSUE TITLE
Feature: Add a person to a starter pack from their profile page

### ISSUE THREAD

**Issue by brandonroberts** [bluesky-social/social-app#6104]
### Describe the Feature

Currently, in order to add someone to a starter pack, you have to go to the starter packs tab on your profile, select the starter pack, edit it, and search for the new person/people you want to add it.

This proposes adding a profile menu option to add a person to starter packs similar to adding a person to lists.

### Attachments

I cloned the repository and prototyped this feature locally, and can put up a full PR if needed.

https://bsky.app/profile/brandonroberts.dev/post/3l7zexpasgl23

https://github.com/user-attachments/assets/1c663650-a721-4f4a-9478-ebc3910d5499

**Comment by bdbch** [comment]
Good idea. In general the UI to add people to an already existing starter kit is weird because I didn't see that I have to click on "Next" to add people. But being able to just add users from their profiles would make a big difference!

**Comment by notque** [comment]
This is a good change, and should be implemented in my view.

**Comment by climatebrad** [comment]
One possibility is to implement it via the "Add to lists" dropdown
<img width="188" alt="Screenshot 2024-11-12 at 9 27 57 AM" src="https://github.com/user-attachments/assets/d72fac22-ee68-4f76-b575-5b74e5ed5308">

**Comment by danielkliu** [comment]
This would be fantastic! Great quality of life feature
